using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -365,6 +423,8 @@ If the given url is nil or empty, it is returned as is. Otherwise, the following
 ACPIdentity.appendVisitorInfoForURL(baseURL);
 ```
 
+o *baseUrl* is the URL to which the visitor info needs to be appended. Returned as is if it is nil or empty.
```

Comment:
- Move to after the code example.
- Reword the 2nd sentence to read: "If the visitor information is nil or empty, the URL is returned as is." and move to the next line.